main.go

```diff
@@ -118,6 +121,19 @@ func main() {
 		}
 	}
 
+	// If the OPERATOR_NAME was not found and the flag leader-election-id was not used, get the operatorName from
+	// the config (valid just for the new layout)
+	if len(strings.TrimSpace(leaderElectionID)) < 1 && kbutil.HasProjectFile() {
+		cfg, err := kbutil.ReadConfig()
+		if err != nil {
+			setupLog.Error(err, "unable to read project config")
+		}
+		leaderElectionID = filepath.Base(cfg.Repo + "-leader-lock")
+	}
+
+	setupLog.Info("setting Manager options:", "MetricsBindAddress", metricsAddr, "LeaderElection",
+		enableLeaderElection, "LeaderElectionID", leaderElectionID, "LeaderElectionNamespace", leaderElectionNamespace)
```

Comment:
we need to have the information since it is very helpful to troubleshooting
Done another PR just within case this suggestion not be accepted. :